vignettes/userguide.Rmd

```diff
@@ -189,6 +189,42 @@ options(
 
 We recommend that you [set up SSH keys](https://www.digitalocean.com/community/tutorials/how-to-configure-ssh-key-based-authentication-on-a-linux-server) for password-less login.
 
+### Options
+
+The various configurable options are mentioned throughout the documentation where applicable, however, we list all of the options here for reference.
+
+Options can be set by including a call to `options()` in your `.Rprofile`, or by calling `options()` in a script or interactively during a session.
+
+- `clustermq.scheduler` 
```

Comment:
Listing: Please use single-level, one space, star as enumerator, no extra newlines to be consistent with the rest of the document (see e.g. [here](https://github.com/mschubert/clustermq/blob/master/vignettes/userguide.Rmd#L207-L209))